cmd/rofl/deploy.go

```diff
@@ -30,6 +31,8 @@ import (
 	cliConfig "github.com/oasisprotocol/cli/config"
 )
 
+const DEFAULT_TESTNET_ROFL_PROVIDER = "oasis1qp2ens0hsp7gh23wajxa4hpetkdek3swyyulyrmz"
```

Comment:
This should be handled via some sort of a defaults map, per network, per paratime. See how it is done for scheduler apps in provider defaults.

Reply:
Something like this:
https://github.com/oasisprotocol/cli/blob/fd6094946bfa1b19ac7fb401faf2ff40f4c091c3/build/rofl/provider/defaults.go#L3-L8